roles/test-win-winlogbeat-file/tasks/main.yml

```diff
@@ -4,13 +4,13 @@
 - name: Install Winlogbeat service
   script: run_script.ps1 -script {{ installdir }}\\install-service-{{ beat_name }}.ps1
 
-- name: Disable elasticsearch output and enable file output
+- name: Replace configuration file with template (windows)
+  win_copy: src={{beat_name}}.yml dest={{installdir}}
+
+- name: Replace template variables
   script: replace.ps1 -file {{ installdir }}\\winlogbeat.yml -find "{{ item.find | quote }}" -replace "{{ item.replace | quote }}"
   with_items:
-    - {find: '(?m)(^\s*)(elasticsearch:)', replace: '$1#$2'}
-    - {find: '(?m)(^\s*)(hosts:.*)', replace: '$1#$2'}
-    - {find: '(?m)(^\s*)#(file:.*)', replace: '$1$2'}
-    - {find: '(?s)(\s+file:.*)#(path:)[^\n]*', replace: '$1$2 {{ winlogbeat_data_dir }}/output'}
+    - {find: '(?s)\{\{ winlogbeat_data_dir \}\}', replace: '{{ winlogbeat_data_dir }}'}
```

Comment:
Hmm, sounds like a broken ansible installation, but the installation on the hetzner machine seems  sane (installed with pip, version 1.9.4). Maybe the win_template module is broken.